### DISCUSSION TITLE
How to search for tagged faces

### DISCUSSION THREAD

**Discussion by protobock** [immich-app/immich#5910] [Q&A] [answered]
Hi,
I am running Immich in an Ubuntu VM on Proxmox.
My library contains now about 50k assets and will grow up to ca 200k. I have spent dozens of hours tagging faces. 
Model is buffalo_l.

Sorry for the silly question but I cannot figure how I can search for a tagged person.
When I type the name of a tagged person in the search bar, it returns 100 assets of various apparently random people, regardless of the searched name. The only apparent relation between the searched name and the output is the gender (If I search for "Enora", I receive 100 female assets and if I search for "Roger", it gives me 100 male pictures).

For information, when I am on the "People" screen and click on a tagged face, the result seems to be absolutly correct.

Thanks in advance for your help

**Comment by bo0tzz** [comment, marked as answer]
This isn't possible at the moment, the search will get improved soon.

**Reply by protobock** [reply]
Thank you so much for the very quick clarification.
